package.json

```diff
@@ -32,11 +35,9 @@
   },
   "devDependencies": {
     "bower": "1.7.9",
+    "cypress": "^1.1.2",
```

Comment:
What about freezing `cypress` version by removing the `^` before the version number?